### PR TITLE
docs(api): remove metadata

### PR DESCRIPTION
**Pull request by matt-primrose**
Removed Metadata API paths
Fixed capitalization issues
fixed some schema issues

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -331,23 +331,67 @@ paths:
...
             application/json:
               schema:
                 type: array
                 items:
                   $ref: '#/components/schemas/Device'
         500:
-          description: 'internal server error'
+          description: 'Internal server error'
+    post:
+      description: Adds a new device to MPS
```
Comment (rsdmike) [inline swagger.yaml:344]: do you wanna mention the requirement to insert into vault separately?

File: swagger.yaml [hunk 2]
```diff
@@ -366,17 +410,20 @@ paths:
...
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/Device'
-  /api/v1/devices/disconnect/{guid}:
+        500:
+          description: 'Internal server error'
     delete:
-      description: Forces a specified device to disconnect from the MPS. Device will reconnect again if it is able.
+      description: Removes the device with a specific GUID and all device information from the database
```
Comment (rsdmike) [inline swagger.yaml:426]: Might want to add that this will also prevent the device from connecting even if a device has not yet been unprovisioned.

**Review by rsdmike — CHANGES_REQUESTED** [review]

**Review by rsdmike — APPROVED** [review]

File: swagger.yaml (hunk 2 above)
Comment (matt-primrose) [reply swagger.yaml:426]: yup, that is new since these changes.  Good catch

File: swagger.yaml (hunk 1 above)
Comment (matt-primrose) [reply swagger.yaml:344]: Yes, will add that.

**Pull request merged**